source/blog/2018-11-29-december-event.md

```diff
@@ -0,0 +1,37 @@
+---
+title: First Annual DecEmber Event!
+author: Melanie Sumner
+tags: Recent Posts, 2018
+responsive: true
+---
+
+December is a great month to support your friendly Ember.js learning team!
+
+Maybe December is a busy month for you, because of the holidays and family obligations- and that’s totally okay- we hope you have a great time! But for some, December is a time of the year with a lot of downtime. Maybe that’s you! Are you wondering what to do with all your vacation time in December? We’ve got ya covered!
```

Comment:
```suggestion
Maybe December is a busy month for you, because of the holidays and family obligations—and that’s totally okay—we hope you have a great time! But for some, December is a time of the year with a lot of downtime. Maybe that’s you! Are you wondering what to do with all your vacation time in December? We’ve got ya covered!
```